{"uuid": "25e31d8d-e5ff-4f82-a1b5-42c64f65a150", "children": ["95279e7b-3ff0-4cd9-99a6-5d2c69762b32"], "befores": [{"name": "exe_driver", "status": "passed", "start": 1679395530201, "stop": 1679395532214}], "afters": [{"name": "exe_driver::0", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"E:\\python\\python3.10.4\\lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"E:\\python\\python3.10.4\\lib\\site-packages\\_pytest\\fixtures.py\", line 917, in _teardown_yield_fixture\n    next(it)\n  File \"E:\\Project\\python\\uiAutoTest\\scripts\\conftest.py\", line 19, in exe_driver\n    GetDriver.quit_web_driver()\n  File \"E:\\Project\\python\\uiAutoTest\\tools\\get_driver.py\", line 29, in quit_web_driver\n    cls.__web_driver.quit()\n  File \"E:\\python\\python3.10.4\\lib\\site-packages\\selenium\\webdriver\\edge\\webdriver.py\", line 71, in quit\n    self.edge_service.stop()\n  File \"E:\\python\\python3.10.4\\lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 151, in stop\n    self.send_remote_shutdown_command()\n  File \"E:\\python\\python3.10.4\\lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 132, in send_remote_shutdown_command\n    if not self.is_connectable():\n  File \"E:\\python\\python3.10.4\\lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 115, in is_connectable\n    return utils.is_connectable(self.port)\n  File \"E:\\python\\python3.10.4\\lib\\site-packages\\selenium\\webdriver\\common\\utils.py\", line 106, in is_connectable\n    socket_ = socket.create_connection((host, port), 1)\n  File \"E:\\python\\python3.10.4\\lib\\socket.py\", line 833, in create_connection\n    sock.connect(sa)\n"}, "start": 1679395547580, "stop": 1679395548649}], "start": 1679395530201, "stop": 1679395548649}